shared/checks/oval/installed_OS_is_wrlinux1019.xml

```diff
@@ -0,0 +1,39 @@
+<def-group>
+  <definition class="inventory" id="installed_OS_is_wrlinux1019" version="1">
+    <metadata>
+      <title>WRLinux 1019</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <reference ref_id="cpe:/o:windriver:wrlinux"
+      source="CPE" />
+      <description>The operating system installed on the system is
+      Wind River Linux 1019</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_wrlinux1019" />
+      <criterion comment="WRLinux is installed" test_ref="test_wrlinux1019" />
+    </criteria>
+  </definition>
+  
+<ind:textfilecontent54_test check="all" check_existence="at_least_one_exists" comment="Check WRLinux10.19 version" id="test_wrlinux1019" version="1">
+    <ind:object object_ref="obj_name_wrlinux1019" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_name_wrlinux1019" version="1" comment="Check WRLinux1019 produce name">
+    <ind:filepath>/etc/os-release</ind:filepath>
+    <ind:pattern operation="pattern match">^NAME=.Wind[\s]+River[\s]+Linux</ind:pattern>
```

Comment:
I don't know what is included in `/etc/os-release`, but if possible, I recommend to improve the pattern to match only WRLinux 1019.
Something like this: https://github.com/ComplianceAsCode/content/blob/69ae8f5a1c696b05beb5f1ab5ccdd5d40e90dac8/shared/checks/oval/installed_OS_is_debian8.xml#L30
or https://github.com/ComplianceAsCode/content/blob/69ae8f5a1c696b05beb5f1ab5ccdd5d40e90dac8/shared/checks/oval/installed_OS_is_ubuntu1604.xml#L22

Reply:
WRLinux1019 use /etc/os-release to record the relevant version message and produce name, I will match more message in /etc/os-release file for WRLinux1019, and submit another version pull request.